modules/oe_paragraphs_media/oe_paragraphs_media.install

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Install methods for OpenEuropa Media Paragraphs.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\field\Entity\FieldConfig;
+
+/**
+ * Implements hook_install().
+ */
+function oe_paragraphs_media_install() {
+  // Add the Text with featured media paragraph to the list of available
+  // paragraphs on the content row paragraph.
+  $field_config = FieldConfig::load('paragraph.oe_content_row.field_oe_paragraphs');
+  $settings = $field_config->get('settings');
+  $settings['handler_settings']['target_bundles']['oe_text_feature_media'] = 'oe_text_feature_media';
+  $settings['handler_settings']['target_bundles_drag_drop']['oe_text_feature_media'] = [
+    'enabled' => TRUE,
+    'weight' => 20,
+  ];
+  $field_config->set('settings', $settings);
+  $field_config->save();
```

Comment:
Let's do this only on EWCMS, the module simply provides a new paragraph but does not assign it to anything.